doc/examples/uncertainty_emcee.ipynb

```diff
@@ -1,205 +1,324 @@
+    "s = ps.EmceeSolve(\n",
+    "    nwalkers=20,\n",
+    "    moves=emcee.moves.DEMove(),\n",
+    "    objective_function=ln_prob,\n",
+    "    progress_bar=True,\n",
+    "    parallel=False,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In the above code we created an `EmceeSolve` instance with 20 walkers, which take steps according to the `DEMove` move algorithm (see Emcee docs), and a Gaussian likelihood function that assumes AR1 correlated errors. Different objective functions are available, see the Pastas documentation on the different options. \n",
+    "\n",
+    "Depending on the likelihood function a number of additional parameters need to be inferred. These parameters are not added to the Pastas Model instance, but are available from the solver object. Using the `set_parameter` method of the solver, these parameters can be changed."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "s.parameters"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "s.set_parameter(\"ln_sigma\", initial=0.0028, vary=False, dist=\"norm\")\n",
+    "s.parameters"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 2c. Run the solver and solve the model\n",
+    "\n",
+    "After setting the parameters and creating a EmceeSolve solver instance we are now ready to run the MCMC analysis. We can do this by running `ml.solve`. We can pass the same parameters that we normally provide to this method (e.g., `tmin` or `fit_constant`). Here we use the initial parameters from our least-square solve, and do not fit a noise model (`noise=False`), because we take autocorrelated errors into account through the likelihood function. \n",
+    "\n",
+    "All the arguments that are not used by `ml.solve`, for example `steps` and `tune`, are passed on to the `run_mcmc` method from the sampler (see Emcee docs). The most important is the `steps` argument, that determines how many steps each of the walkers takes."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Use the solver to run MCMC\n",
+    "ml.solve(\n",
+    "    solver=s,\n",
+    "    initial=False,\n",
+    "    fit_constant=False,\n",
+    "    noise=False,  # We have to set noise to False !\n",
+    "    tmin=\"1990\",\n",
+    "    steps=1000,\n",
+    "    tune=True,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 3. Posterior parameter distributions\n",
+    "\n",
+    "The results are stored in the `result` object, accessible through `ml.fit.sampler` variable. The object `ml.fit.sampler.flatchain` contains a Pandas DataFrame with $n$ the parameter samples, where $n$ is calculated as follows:\n",
```

Comment:
The `result` object? Or is that what emcee calls it?

Reply:
this should have been the sampler object, changed.